insights/client/apps/ansible/playbook_verifier/__init__.py

```diff
@@ -132,6 +132,8 @@ def verify(playbook, checkVersion=True, skipVerify=False):
     # Egg Version Check
     eggVersioningCheck(checkVersion)
```

Comment:
IMO if we're wholesale skipping playbook verification til it's ready, we might as well skip the version check as well so the function never fails.